### ISSUE TITLE
Glomerular proteinuria

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6399]
**Preferred term label:**
Glomerular proteinuria

**Synonyms**

**Definition (free text, please give PubMed ID)**
Proteinuria wih an increased permeability of the glomerular capillary wall to macromolecules (particularly albumin). 
PMID: 19634399

**Parent term (use hpo.jax.org/app)**
Proteinuria

**Comment by pnrobinson** [comment]
@avizrosenberg @yongqunh -- can you please comment?

@johannaest  -- the HPO does not encode pathophysiology -- it is limited to observable phenotypes. I do not think that if you are measuring protein in urine it is clearly possible to differentiate, except that severe proteinuria is usually glomerular. The HPO currently has terms for mild, moderate, severe proteinuria, and I believe these are the actual phenotypic features (observations).

Therefore, probably this is not a good candidate for an HPO term, but if I am missing something please respond!

**Comment by yongqunh** [comment]
I feel that it's difficult to define "observable phenotypes". For example, the lab test results cannot be observed by eyes directly and will need to be tested using lab test. Similarly, protein levels cannot be observed directly but can by some experimental test. I think it's probably fine to have encode pathophysiological phenotypes, depending on how to define "observable".

**Comment by johannaest** [comment]
​Hello everyone,



I agree with Oliver. There are already other HPO terms that are not phenotypic observable for example Methemoglobulinemia. This is a lab test result. It is difficult to differentiate here.​



Best,

Johanna


 




Von: Yongqun Oliver He <notifications@github.com>
Gesendet: Mittwoch, 2. Dezember 2020 20:01
An: obophenotype/human-phenotype-ontology
Cc: Est, Johanna Ligaya; Mention
Betreff: Re: [obophenotype/human-phenotype-ontology] Glomerular proteinuria (#6399)
 



I feel that it's difficult to define "observable phenotypes". For example, the lab test results cannot be observed by eyes directly and will need to be tested using lab test. Similarly, protein levels cannot be observed directly but can by some experimental
 test. I think it's probably fine to have encode pathophysiological phenotypes, depending on how to define "observable".
—
You are receiving this because you were mentioned.
Reply to this email directly, 
view it on GitHub, or 
unsubscribe.

**Comment by pnrobinson** [comment]
@yongqunh @johannaest  -- by observable I mean measurable, not necessarily observable with our eyes.

There are phenotypic differences between tubuluar and glomerular proteinuria, e.g. 
Unlike glomerular proteinuria, where protein excretion can reach 20 g/24 h and consists mainly of albumin, tubular proteinuria is generally < 1–2 g/24 h. In tubular proteinuria, although albumin remains a significant component, there is a relatively much greater increase in proteins of < 60 000 Da 
@avizrosenberg -- is there a recommendation about how to classify proteinuria like this?

**Comment by avizrosenberg** [comment]
the major difference between glomerular and tubular protein is in the composition of the protein. One typically has a significant albumin component in glomeuralr proteinuria compared with b2 Microglobulin in tubular protein. This is all in contrast to other phenotypes wherein there may be a paraprotein (i.e., light chains)

**Comment by yongqunh** [comment]
There is a typo in the definition:
"Proteinuria wih an increased permeability" --> "Proteinuria with an increased permeability "
otherwise, looks good to me.

The definition appears to be aligned with the description in the paper (PMCID: PMC4953623, PMID: 19634399)
copied:
"Glomerular proteinuria
Glomerular disease causes an increase in the permeability of the glomerular capillary wall to macromolecules (particularly albumin) and results in glomerular proteinuria. It is persistent and may be associated with haematuria and reduced GFR." 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4953623/

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC322340/

**Comment by yongqunh** [comment]
It appears to be a good reference!

**Comment by pnrobinson** [comment]
added
